mysite/settings.py

```diff
@@ -106,16 +110,27 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'Asia/Kolkata'
 
 USE_I18N = True
 
 USE_L10N = True
 
-USE_TZ = True
+USE_TZ = False
```

Comment:
changed datetime.now() to timezone.now() everywhere to and have read about differences in them and what are naive and aware datimetime
But Keeping USE_TZ =True and timezone.now() will store time in 'UTC' in db in case of postgres irrespective of the timezone specified.
But browser displays the time according to machine.